WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -196,6 +197,7 @@ extension ReaderTabViewController {
         static let encodedIndexKey = "WPReaderTabControllerIndexRestorationKey"
         static let discoverIndex = 1
         static let spotlightOffset = UIOffset(horizontal: 20, vertical: -10)
+        static let settingsButtonContentEdgeInsets = UIEdgeInsets(top: 2, left: 0, bottom: 0, right: 0)
```

Comment:
Tweak the icon offset to align it with the Search 🔍 icon
| Before | After |
| ------- | ----- |
| <img width="439" src="https://user-images.githubusercontent.com/9609223/223441331-14553a4f-aba6-443d-9339-f2ba85c27e5b.png"> | <img width="438" src="https://user-images.githubusercontent.com/9609223/223441426-5249769e-d35e-470d-a3e7-78442534899b.png"> |

Reply:
The slight move downwards looks good!